LoginPage.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Formik, Field, Form } from 'formik';
+import { useHistory } from 'react-router-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const LoginPage = () => {
+  const history = useHistory();
+
+  return (
+    <div className="container-fluid">
+      <div className="row">
```

Comment:
Consider extracting the image source and alt text to a constant or prop for better maintainability and reusability.